src/eve-server/pos/Structure.cpp

```diff
@@ -440,7 +453,7 @@ void StructureSE::Init()
      */
     if (m_data.timestamp > 0)
     {
-        // do something constructive here.
+        m_data.timestamp = GetFileTimeNow();
```

Comment:
I don't think we need to set this here since the timestamp being zero indicates that it has just been constructed and hasn't gone through a Process() cycle yet. It will get its timestamp once it proceeds with its first loop of Process() and various other functions. 

Any thoughts @Almamu?

Reply:
Right, it IS kind of a placeholder, container if note anchored should desapear After 2day without interaction so i will need the last interaction timestamp. Will clean it out.